.github/workflows/terraform.yaml

```diff
@@ -0,0 +1,28 @@
+name: Check the Terraform Module
+
+on:
+  workflow_call:
```

Comment:
- I saw elsewhere a filter with *.tf files. I assume here too we want this to be conditional on the presence of terraform files?
- Looks like this action assumes the relevant tf files at the root. Should we have an input for root folder?

Reply:
I added path to perform the checks only for *.tf files.